app/assets/scripts/map.js

```diff
@@ -46,38 +53,64 @@ function initMap() {
           place.address_components[2].short_name) || '')
       ].join(' ');
     }
-    marker.setLabel(address);
+
+    // create custom location marker
+    var marker = new google.maps.Marker({
+      map: map,
+      position: place.geometry.location,
+      anchorPoint: new google.maps.Point(0, -29),
+      icon: {
+        path: google.maps.SymbolPath.CIRCLE,
+        scale: 8,
+        strokeColor: 'grey',
+        fillColor: 'black',
+        fillOpacity: 1,
+      },
+    });
     var placeDiv = document.createElement('div');
     var br = document.createElement('br');
     $(placeDiv).append(place.name, br, address);
     infowindow.setContent(placeDiv);
     infowindow.open(map, marker);
   });
+}
 
-  var marker = new google.maps.Marker({
-    map: map
+function initResourceSearch() {
+  $('#resources-form').submit(function(e) {
+    e.preventDefault();
```

Comment:
The event e - this was added by sanjay (git thinks i added a lot of these things when I just move their location in the file). This is to prevent the browser from jumping url